app/controllers/games_controller.rb

```diff
@@ -29,9 +27,9 @@ def create
     end
   end
 
-  # POST '/submit_move/:move'
+  # POST '/submit_move/:id/:move'
   def submit_move
-    @game = Game.find(1)
+    @game = Game.find(params[:id])
```

Comment:
can we call the private method `set_game` or add it to the before hook above?

Reply:
Yup will fix